packages/dds/sequence/src/test/snapshots/legacy/withIntervals.json

```diff
@@ -5,7 +5,7 @@
     "encoding": "utf-8"
```

Comment:
Doesn't have to be this PR, but before finishing up this work we should modify this test suite to accommodate "formats we support reading but don't write" to lock in the partial support we have for this old format. Ideally we'd like clear test failures if someone tries to remove the back-compat handling in IntervalCollection's constructor

Reply:
Created [829](https://dev.azure.com/fluidframework/internal/_workitems/edit/829) to track this.